spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/plugin/GravitinoDriverPlugin.java

```diff
@@ -69,6 +73,17 @@ private void registerGravitinoCatalogs(SparkConf sparkConf, Set<String> catalogN
         });
   }
 
-  // Todo inject Iceberg extensions
-  private void registerSqlExtensions() {}
+  private void registerSqlExtensions(SparkConf conf) {
```

Comment:
could you remove the duplicated extension between setting in config and `GRAVITINO_DRIVER_EXTENSIONS`?

Reply:
updated.